server/services/cohorts.ts

```diff
@@ -4,7 +4,7 @@ import utc from 'dayjs/plugin/utc';
 import { dbClient, collections } from '../db';
 import { validate, CreateCohortSchema } from '../validation';
 import { getPostHireStatusesForParticipant } from './post-hire-flow';
-import { postHireStatuses } from '../constants';
+import { postHireStatuses, ParticipantStatus as ps } from '../constants';
```

Comment:
Is this just to avoid confusing things with the Massive queries?

Reply:
It's just to make sure we're using our constants as much as possible, instead of leaving the string equivalent of magic numbers everywhere.